students/barecki_andrzej/lesson_05_lists/swap_min_and_max.py

```diff
@@ -0,0 +1,8 @@
+input_list = input().split(' ')
+for row in range(0, len(input_list)):
+    input_list[row] = int(input_list[row])
+min_index = input_list.index(min(input_list))
+max_index = input_list.index(max(input_list))
+input_list[min_index], input_list[max_index] = \
```

Comment:
Jeżeli zmienną nazywamy `input_list` to nie powinniśmy przechowywać w niej wyników kolejnych operacji, bo jest to mylące dla czytającego. Powinna być nowa zmienna, w której wyląduje wynik.

Nie wymagam poprawki.